Source/WarpX.H

```diff
@@ -187,6 +187,13 @@ public:
     // do nodal
     static int do_nodal;
 
+    std::array<const amrex::MultiFab*, 3> get_array_Bfield_aux  (int lev) const { return {Bfield_aux[lev][0].get(),
+                                                                                          Bfield_aux[lev][1].get(),
+                                                                                          Bfield_aux[lev][2].get()}; }
+    std::array<const amrex::MultiFab*, 3> get_array_Efield_aux  (int lev) const { return {Efield_aux[lev][0].get(),
+                                                                                          Efield_aux[lev][1].get(),
+                                                                                          Efield_aux[lev][2].get()}; }
```

Comment:
These lines are a bit too long to read well on GitHub for reviews:
```suggestion
    std::array<const amrex::MultiFab*, 3>
    get_array_Bfield_aux(const int lev) const
    {
        return {
            Bfield_aux[lev][0].get(),
            Bfield_aux[lev][1].get(),
            Bfield_aux[lev][2].get()
        };
    }

    std::array<const amrex::MultiFab*, 3>
    get_array_Efield_aux(const int lev) const
    {
        return {
            Efield_aux[lev][0].get(),
            Efield_aux[lev][1].get(),
            Efield_aux[lev][2].get()
        };
    }
```

Maybe we should also add a doxygen string here?

Reply:
Ok I'll try some indentation and add a docstring.